Clever open : This vhost doesn't exist

After a `clever open` I get this message:

```
[ERROR] This vhost doesn't exist
```

I tested it on two systems. I reproduce with existing and just created apps.

--- a/src/models/domain.js
+++ b/src/models/domain.js
@@ -11,7 +11,7 @@ async function getBest (appId, orgaId) {
   return getFavouriteDomain({ id: orgaId, appId }).then(sendToApi)
     .catch(async (e) => {
 
-      if (e.status !== 404) {
+      if (e.response.status !== 404) {
         throw e;
       }
 

